kahuna/public/stylesheets/main.css

```diff
@@ -1430,12 +1431,22 @@ FIXME: what to do with touch devices
     overflow: hidden;
 }
 
+.image-info__group--fixed-panel .labeller{
```

Comment:
Or are we avoiding that as we wold either have to create another view or send over a var as a state variable to the directive?